lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -7,6 +7,7 @@ module Engine
     module Analyzers
       module Ruby
         class Main
+          DEFAULT_PATHS = ["**/*.rb", "**/*.rake", "Rakefile"]
```

Comment:
I haven't even used all those different Ruby libs, and, yeah, I bet some of them would be nonsensical to check for duplication. I think going through that list and using your own discretion is probably best.